docs/FAQ.md

```diff
@@ -25,9 +25,6 @@ by this command is very limited and you may want to look for other tools if you
 There are many excellent serial terminals to chose from. On Linux or macOS, you may already have [screen][screen]
 installed. On Windows, a good choice for command line usage is Plink, included with [PuTTY][putty].
 
-Arduino CLI does provide a gRPC interface which offers the capability for powerful integration with custom monitors. See
-the [Monitor service documentation][monitor service].
```

Comment:
Is this still required? https://github.com/arduino/arduino-cli/blob/899dc91b3e2e12948badaffc25eca2cfaefa2eda/docs/FAQ.md?plain=1#L40

Reply:
We should replace this one earlier, I'm preparing a PR for this one only.